### PR TITLE
design_2

### PR DESCRIPTION
**Pull request by KikiGarthwaite**
Login Controller:
- Increased height of inputs container on register
- Capitalized “name” in filler text
- Adjusted colors of segmented control

Chat Log Controller:
- Thickened separator line
- Changed input container to corner radius 10 for consistency
- Tried to implement scrolling to bottom of page —> not working right now but code is not breaking anything

Chat Message Cell:
- Changed bubbles to Stride blue
- Changed corner radius of message bubbles to 10
- Disabled user from editing chat message bubble

User Cell:
- Bolded username on messages view
- Added border color to profile picture
- Changed color of time label
- Edited time label constraints 

Edit Profile Controller & Edit Password Controller:
- Bolded label text
- Changed label text go blue
- Changed view background color to light gray
- Added space between separator line view and right side of the screen
- Added grade and school input containers

Profile Page:
- Completely redesigned profile page

New Message Controller:
- Added section headers
- Added drop shadow to section headers
- ***Problem: headers are there but user not being sorted into correct header sections… every user is listed under each section header right now

### CODE REVIEW

File: StrideLearningApp/View Controllers/EditPasswordController.swift [hunk 1]
```diff
@@ -97,7 +103,8 @@ class EditPasswordController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        view.backgroundColor = .white
+        //view.backgroundColor = .white
```
Comment (JordanDenning) [inline StrideLearningApp/View Controllers/EditPasswordController.swift]: can you just delete this line so we keep commented code out

File: StrideLearningApp/View Controllers/EditProfileController.swift [hunk 2]
```diff
@@ -94,7 +148,8 @@ class EditProfileController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        view.backgroundColor = .white
+        //view.backgroundColor = .white
```
Comment (JordanDenning) [inline StrideLearningApp/View Controllers/EditProfileController.swift]: this line too

File: StrideLearningApp/View Controllers/LoginController.swift [hunk 3]
```diff
@@ -233,12 +233,14 @@ class LoginController: UIViewController {
     lazy var loginRegisterSegmentedControl: UISegmentedControl = {
         let sc = UISegmentedControl(items: ["Login", "Register"])
         sc.translatesAutoresizingMaskIntoConstraints = false
-        sc.tintColor = UIColor(r: 238, g: 238, b: 238)
+        //sc.backgroundColor = UIColor(r: 238, g: 238, b: 238)
```
Comment (JordanDenning) [inline StrideLearningApp/View Controllers/LoginController.swift]: and this line

File: StrideLearningApp/View Controllers/MessagesController.swift [hunk 4]
```diff
@@ -50,6 +50,8 @@ class MessagesController: UITableViewController, UISearchResultsUpdating, UISear
         
         tableView.delegate = self
         tableView.dataSource = self
+        //tableView.separatorColor = UIColor(r:16, g:153, b:255)
+
```
Comment (JordanDenning) [inline StrideLearningApp/View Controllers/MessagesController.swift]: and these two lines

File: StrideLearningApp/View Controllers/MessagesController.swift [hunk 5]
```diff
@@ -71,6 +73,7 @@ class MessagesController: UITableViewController, UISearchResultsUpdating, UISear
         searchController.searchBar.placeholder = "Search..."
         searchController.searchBar.delegate = self
         searchController.searchBar.sizeToFit()
+        //searchController.searchBar.barTintColor = UIColor(r: 16, g: 153, b: 255)
```
Comment (JordanDenning) [inline StrideLearningApp/View Controllers/MessagesController.swift]: and this line

File: StrideLearningApp/View Controllers/NewMessageController.swift [hunk 6]
```diff
@@ -68,6 +72,8 @@ class NewMessageController: UITableViewController, UISearchResultsUpdating, UISe
         searchController.searchBar.sizeToFit()
         tableView.tableHeaderView = searchController.searchBar
     }
+    
+
```
Comment (JordanDenning) [inline StrideLearningApp/View Controllers/NewMessageController.swift]: and these two blank lines. better to have code condensed and have unnecessary white space

File: StrideLearningApp/View/UserCell.swift [hunk 7]
```diff
@@ -37,6 +37,8 @@ class UserCell: UITableViewCell {
                 
                 if let dictionary = snapshot.value as? [String: AnyObject] {
                     self.textLabel?.text = dictionary["name"] as? String
+                    self.textLabel?.font = UIFont.boldSystemFont(ofSize: 16)
+                    //self.textLabel?.textColor = UIColor(r: 16, g:153, b:255)
```
Comment (JordanDenning) [inline StrideLearningApp/View/UserCell.swift]: delete

**Review by choybm — COMMENTED** [review]
Looks good on my end!

File: StrideLearningApp/View/UserCell.swift [hunk 8]
```diff
@@ -67,8 +71,9 @@ class UserCell: UITableViewCell {
     let timeLabel: UILabel = {
         let label = UILabel()
         //        label.text = "HH:MM:SS"
```
Comment (JordanDenning) [inline StrideLearningApp/View/UserCell.swift]: delete

File: StrideLearningApp/View/UserCell.swift [hunk 9]
```diff
@@ -67,8 +71,9 @@ class UserCell: UITableViewCell {
...
-        label.font = UIFont.systemFont(ofSize: 13)
-        label.textColor = UIColor.darkGray
+        label.font = UIFont.systemFont(ofSize: 12)
+        //label.textColor = UIColor.darkGray
```
Comment (JordanDenning) [inline StrideLearningApp/View/UserCell.swift]: delete

**Review by JordanDenning — CHANGES_REQUESTED** [review]
just some lines to delete if we're not using them anymore to keep code clean

**Pull request closed without merging**